lib/offlinetraces.js

```diff
@@ -98,22 +138,26 @@ module.exports = function (kafkaConfig) {
                 // './offlinetraces/' + req.params.activityId + '/' + filename;
                 req.params.filename = offlinetrace._id.toString();
 
-                return uploadPromise(req, res);
+                return uploadPromise(req, res, kahoot);
             })
             .then(function (err) {
                 if (err) {
                     throw err;
                 }
 
                 if (!req.file) {
+                    if (kahoot) {
+                        nofileMessage = 'You must upload a .csv file with the traces in the correct CSV format';
```

Comment:
If it's kahoot and there's no file, should you upload a .xlsx.... not a .csv... no?